Intelipro/reja
# Fix: Rename stores to branches

#### What does this PR do?
Rename the stores to branches

#### Description of Task to be completed?
- [x] Rename stores to branches
- [x] Change /store URL to /branch

#### How should this be manually tested?
- Clone this repo `git clone  [the-repo-link]`
- Checkout to this branch `bg-fix-store-changes`
- Install all dependencies `npm install`
- Start the server `npm run start`
-  Log in to the application and go to upload-section. Click the dropdown `Choose available data` you'll see the name Branch

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?

#### Screenshots (if appropriate)

#### Questions:
